recipes/pandavro/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pandavro" %}
+{% set version = "1.5.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: fd4ede1600cfc08150f28a7a1c1dd4abfbc87d76cd4a5b1639c8f55a04e97914
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*
+    - pip
+  run:
+    - python >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*
+    - fastavro >=0.14.11
+    - numpy >=1.7.0
+    - pandas >=1.0.0
+    - six >=1.9
+
+test:
+  imports:
+    - pandavro
+
+about:
+  home: https://github.com/ynqa/pandavro
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```

Comment:
Since you are manually packaging the license file, please add the following comment:

```
...
# License file is manually packaged. Please see: https://github.com/ynqa/pandavro/pull/23
license_file: ...
```

The link to https://github.com/ynqa/pandavro/pull/23 is to act as a reminder that in future releases you should check to see if the license file is included in the source distribution. 

Also note that in the upstream repo, the license file is called 'LICENSE` rather than `LICENSE.txt`, so if the opportunity arises to remove the copy in the feedstock, you'll have to change the file name unless you rename the file here.

Reply:
I renamed the license file and added the comment.